src/styled/styled.tsx

```diff
@@ -9,32 +9,30 @@ const customConfig = windowObj?.__STYLED_WIND_CUSTOM_CONFIG__ || {};
 const config = generateStylesJS(customConfig);
 const cssGen = new CSSGen(config);
 
-const getHydratedTemplateString = (strings: TemplateStringsArray) => {
+const getHydratedTemplateString = (strings: TemplateStringsArray): any => {
```

Comment:
@jyash97 this looks great, can you please separate this into a util function and write test for the same, so that this won't regress. Also add comment in that test to point to this particular issue.

Reply:
Done 👍